src/styles/index.scss

```diff
@@ -1,3 +1,221 @@
+
 body {
   margin: 0;
+  font-family: $font-family;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: $font-family;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $tertiary-color;
+  transition: color 0.3s;
+
+  &:hover {
+    color: $primary-color;
+  }
+
+  &.is-active {
+    color: $primary-color;
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      height: 4px;
+      border-radius: 2px;
+      background-color: $primary-color;
+    }
+  }
+}
+
+.main {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.main__catalog {
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  align-items: center;
+  align-content: center;
+  grid-template-rows: auto;
+
+  @media screen and (max-width: 487px) {
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  @media screen and (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media screen and (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media screen and (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  justify-content: center;
+  width: 166px;
+  padding: 32px 16px 16px;
+  display: flex;
+  transition: all 300ms linear;
+  max-width: 200px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__price {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+    margin-top: 24px;
+    margin-bottom: 16px;
+    display: flex;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-amount {
+      color: $tertiary-color;
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__name > p {
+    color: $tertiary-color;
+    margin: 0;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    margin: 4px 0 16px;
```

Comment:
You tend to use margin-bottom here to position elements within the card. 
It's a valid approach, but let's keep it consistent by adding margin-bottom to the `card__name` element instead of setting top margin here.